Clinical-Genomics/scout
# Check case (_id, display_name) and case individuals(individual_id, display_name, phenotype) on case reupload

Fix #2723 and fix #2771, check also #2702

Make sure that when a case is re-uploaded it has:
- Same display_name
- Same individuals (cheked keys: individual_id, display_name, phenotype)

Otherwise raise Error.


**How to test**:
1. Load the demo case into a local instance -> `scout setup demo`
2. Modify the demo config file by changing case display name (family_name in config file) and try to update the case with the command ->  `scout --demo load case scout/demo/643594.config.yaml -u` . It should trigger an error
3. Modify the demo config file by changing one of the samples (individual_id, display_name, phenotype) and try to update the case with the command above. It should also trigger error.


**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] tests executed by CR, pytest
